### PR TITLE
feat: archive document page

### PR DESCRIPTION
**Pull request by Ibwedagama**
### Related
- [S11A3 - Download Arsip dan Dokumen](https://app.clickup.com/t/2tyz34b)
- [S11A2 - View Detail Arsip dan Dokumen](https://app.clickup.com/t/2tyz2p8)
- [S11A4 - Search Arsip dan Dokumen](https://app.clickup.com/t/2tyz34r)
- [S11A1 - View List Arsip dan Dokumen](https://app.clickup.com/t/2tyyzny)

### Changes
- feat: add view list document
- feat: add document Filter by topics
- feat: add document search 
- feat: add document detail
- feat: add document download

### Preview
**Mobile**

https://user-images.githubusercontent.com/33661143/187374032-62daba6f-0de4-4068-bac4-84a1d7a505fe.mov

**Tablet**

https://user-images.githubusercontent.com/33661143/187374044-269e4158-fc57-4e49-ba44-3b8e00be979e.mov

**Dekstop**

https://user-images.githubusercontent.com/33661143/187374168-84150938-4a96-4236-928b-b12983b8177d.mov

### Evidence
title: feat archive document page
project: Portal Jabar
participants: @Ibwedagama

### CODE REVIEW

File: components/Document/EmptySearch/index.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        Cobalah untuk menggunakan kata kunci yang berbeda
+      </p>
+    </section>
+    <section v-else class="text-center">
+      <p class="font-lora font-bold text-base leading-relaxed text-gray-800 text-center mb-4">
+        Belum ada laporan untuk kategori ini
+      </p>
+      <p class="font-lato text-sm leading-6 text-gray-700 max-w-[40ch]">
+        Saat ini belum ada laporan yang ada pada kategori ini,
+        silahkan menjelajahi kategori lainya.
```
Comment (yoslie) [inline components/Document/EmptySearch/index.vue:27]: minor wording: maybe its `silakan` not `silahkan`

**Review by yoslie — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

File: components/Document/EmptySearch/index.vue (hunk 1 above)
Comment (naufalihsank) [reply components/Document/EmptySearch/index.vue:27]: i saw this `silahkan` wording issue quite a few times now, maybe we should forward this issue to UI/UX team 🤔

Comment (Ibwedagama) [reply components/Document/EmptySearch/index.vue:27]: thank you @yoslie @naufalihsank , this is always happen when I didn't check the copy 😅 🙏🏼

I agree this issues should be forwarded to UI/UX team

Comment (Ibwedagama) [reply components/Document/EmptySearch/index.vue]: changes [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/316/commits/47713cf65987aa141d5c007313b61fce7adbf752)

thanks

**Pull request merged**